<!--
  Driver for the logitech rumblepad2 joystick.

  Use the D pad while pressing and holding the left trigger button (5) to control.
-->
<launch>
  <!--  enable sound effects -->
  <!--<incude file="$(find sound_play)/soundplay_node.launch">-->
  <!--  smooths inputs from cmd_vel_mux/input/teleop_raw to cmd_vel_mux/input/teleop -->
  <include file="$(find atrv_mini_pkg)/launch/include/velocity_smoother.launch"/>
  
  <node pkg="joy" type="joy_node" name="joystick" output="screen"/>
  <node pkg="atrv_mini_pkg" type="atrv_mini_joy" name="ATRVmini_joy" output="screen">
    <param name="scale_angular" value="0.3"/>
    <param name="scale_linear" value="0.2"/>
    <remap from="ATRVmini_joy/cmd_vel" to="/ATRVmini_node/cmd_vel"/>
  </node>
</launch>
